### PR TITLE
Remove primary key from an example of CREATE SOURCE as source doesn't support it

### LINKED ISSUE #2005: Bug: CREATE SOURCE does not support primary key
### Where's the bug?

https://docs.risingwave.com/docs/current/sql-create-source/#upsert-json

### Describe the bug

e.g. 
 I noticed that there is a [typo/misused term/inconsistency/ambiguity] in [this sentence/paragraph/figure/section/screenshot]. 
 [This] is incorrect because [...]. 
 I think you folks should replace [this] with [that]. 
 If you take a look at [this reference] you will agree with me. 
 Cheers.

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2005

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by xxchan — APPROVED** [review]
Thanks!

BTW, maybe we can also mention at the beginning that if we need primary key (which is required by some formats like `FORMAT UPSERT/DEBEZIUM`), we have to use `CREATE TABLE`. This might help reduce some users' confusion.

image

**Comment by xxchan** [comment]
> which is required by some formats like `FORMAT UPSERT/DEBEZIUM`

Wait, do all formats except `FORMAT PLAIN` need primary key? (in other words, only `FORMAT PLAIN` can be append-only?) @st1page @xiangjinwu

**Comment by hengm3467** [comment]
@ShanlanLi I forgot to mention, as this is a bug, we should make the same doc updates to three previous versions (v1.7, v1.6, and v1.5).

**Comment by xxchan** [comment]
I think this part is also wrong. For `UPSERT` we have to use `TABLE`
image

**Comment by ShanlanLi** [comment]
> @ShanlanLi I forgot to mention, as this is a bug, we should make the same doc updates to three previous versions (v1.7, v1.6, and v1.5).

Yes, I've checked them. This issue just appears in 1.8. Maybe this part has just been added.

**Review by st1page — APPROVED** [review]

**Comment by tabVersion** [comment]
> > which is required by some formats like `FORMAT UPSERT/DEBEZIUM`
> 
> Wait, do all formats except `FORMAT PLAIN` need primary key? (in other words, only `FORMAT PLAIN` can be append-only?) @st1page @xiangjinwu

I think so.

**Review by tabVersion — APPROVED** [review]

**Review by WanYixian — APPROVED** [review]
LGTM

File: versioned_docs/version-1.7/sql/commands/sql-create-source.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ slug: /sql-create-source
 </head>
 
 A source is a resource that RisingWave can read data from. You can create a source in RisingWave using the `CREATE SOURCE` command.
-If you choose to persist the data from the source in RisingWave, use the `CREATE TABLE` command with connector settings. For more details, see [CREATE TABLE](sql-create-table.md).
+
+If you choose to persist the data from the source in RisingWave, use the `CREATE TABLE` command with connector settings. Or if you need to create the primary key (which is required by some formats like FORMAT UPSERT/DEBEZIUM), you have to use `CREATE TABLEF` too. For more details, see [CREATE TABLE](sql-create-table.md).
```
Comment (xiangjinwu) [inline versioned_docs/version-1.7/sql/commands/sql-create-source.md:13]: typo: `TABLEF` -> `TABLE`

File: versioned_docs/version-1.7/sql/commands/sql-create-source.md [hunk 2]
```diff
@@ -83,7 +84,7 @@ To ingest data in formats marked with "T", you need to create tables (with conne
 
 | Connector | Version | Format |
 |---------|---------|---------|
-|[Kafka](/ingest/ingest-from-kafka.md)|3.1.0 or later versions |[Avro](#avro), [JSON](#json), [protobuf](#protobuf), [Debezium JSON](#debezium-json) (T), [Debezium AVRO](#debezium-avro) (T), [DEBEZIUM_MONGO_JSON](#debezium-mongo-json) (T), [Maxwell JSON](#maxwell-json) (T), [Canal JSON](#canal-json) (T), [Upsert JSON](#upsert-json), [Upsert AVRO](#upsert-avro), [Bytes](#bytes)|
+|[Kafka](/ingest/ingest-from-kafka.md)|3.1.0 or later versions |[Avro](#avro), [JSON](#json), [protobuf](#protobuf), [Debezium JSON](#debezium-json) (T), [Debezium AVRO](#debezium-avro) (T), [DEBEZIUM_MONGO_JSON](#debezium-mongo-json) (T), [Maxwell JSON](#maxwell-json) (T), [Canal JSON](#canal-json) (T), [Upsert JSON](#upsert-json) (T), [Upsert AVRO](#upsert-avro), [Bytes](#bytes)|
```
Comment (xiangjinwu) [inline versioned_docs/version-1.7/sql/commands/sql-create-source.md:87]: Upsert AVRO as well

**Pull request merged**